features/tracker_board_view/lib/src/composities/mobile_tracker_board_composite.dart

```diff
@@ -1,123 +1,104 @@
-                  currentHours:
-                      state.timeFrameFilter?.study.current.toString() ?? '',
-                  previousHours:
-                      state.timeFrameFilter?.study.previous.toString() ?? '',
-                  periodTimeType: state.periodTimeType,
-                ),
-                const SizedBox(height: AppDimensions.padding24),
-                MobileTrackerCard(
-                  trackerCardType: TrackerCardType.exercise,
-                  currentHours:
-                      state.timeFrameFilter?.exercise.current.toString() ?? '',
-                  previousHours:
-                      state.timeFrameFilter?.exercise.previous.toString() ?? '',
-                  periodTimeType: state.periodTimeType,
-                ),
-                const SizedBox(height: AppDimensions.padding24),
-                MobileTrackerCard(
-                  trackerCardType: TrackerCardType.social,
-                  currentHours:
-                      state.timeFrameFilter?.social.current.toString() ?? '',
-                  previousHours:
-                      state.timeFrameFilter?.social.previous.toString() ?? '',
-                  periodTimeType: state.periodTimeType,
-                ),
-                const SizedBox(height: AppDimensions.padding24),
-                MobileTrackerCard(
-                  trackerCardType: TrackerCardType.selfCare,
-                  currentHours:
-                      state.timeFrameFilter?.selfCare.current.toString() ?? '',
-                  previousHours:
-                      state.timeFrameFilter?.selfCare.previous.toString() ?? '',
-                  periodTimeType: state.periodTimeType,
-                ),
-                TextButton.icon(
-                  onPressed: () {
-                    AuthService().signOut();
-                  },
-                  icon: const Icon(
-                    Icons.logout,
-                    color: Colors.white,
-                  ),
-                  label: Text(
-                    'general.logout'.tr(),
-                    style: AppTextStyle.rubicRegular20,
+        child: Column(
+          children: <Widget>[
+            MobileProfileMenu(
+              avatar: const AssetImage(AppImages.jeremy),
+              firstName: user.fullName,
+              lastName: 'Popov',
+              selectedPeriodTimeType: state.periodTimeType,
+              onDailyPressed: () {
+                BlocProvider.of<TrackerBoardBloc>(context, listen: false)
+                    .add(const PressDailyButton());
+              },
+              onMonthlyPressed: () {
+                BlocProvider.of<TrackerBoardBloc>(context, listen: false)
+                    .add(const PressMonthlyButton());
+              },
+              onWeeklyPressed: () {
+                BlocProvider.of<TrackerBoardBloc>(context, listen: false)
+                    .add(const PressWeeklyButton());
+              },
```

Comment:
Make a void function something like void _onBarPressed(param){ BlocProvider.of<TrackerBoardBloc>(context, listen: false).add(const PressWeeklyButton());}. Where event depends on this param. And call onMonthlyPressed: () => _onBarPressed(param) for ex.